.travis.yml

```diff
@@ -10,7 +10,10 @@ addons:
   postgresql: "9.3"
 before_script:
   - gem install pg
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "CREATE USER ox_exchange WITH CREATEDB PASSWORD 'ox_exchange'" -U postgres
+  - psql -c 'CREATE DATABASE ox_exchange_test' -U postgres
+  - psql -c 'GRANT ALL PRIVILEGES ON DATABASE ox_exchange_test TO ox_exchange' -U postgres
+  - RAILS_ENV=test bundle exec rake db:migrate
 script:
   - RAILS_ENV=test bundle exec rake --trace
 branches:
```

Comment:
Have a look at this conversation and see what you think: https://github.com/openstax/accounts/pull/176#discussion_r29604742